dev/benchmarks/microbenchmarks/lib/stocks/build_bench_profiled.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Is this file intended as part of this change?

Reply:
Nope, was on the wrong base.  Fixed.